tests/requirements.txt

```diff
@@ -1,2 +1,4 @@
-git+https://github.com/jina-ai/jina.git@v2.0.0rc7#egg=jina[http]
-pytest==6.2.4
\ No newline at end of file
+git+https://github.com/jina-ai/jina.git@v2.0.0rc9#egg=jina[http]
```

Comment:
Why not `jina[http]==2.0.0rc9` ?

Reply:
are they the same?

Reply:
Sorry, my bad, we are actually supposed to do it like you did.